src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -128,8 +131,8 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
     private static validateThroughput(input: string): string | undefined | null {
         try {
             const value = Number(input);
-            if (value < 2500 || value > 100000) {
-                return "Value needs to lie between 2500 and 100,000"
+            if (value < minThroughput || value > maxThroughput) {
+                return `Value needs to lie between ${minThroughput} and ${maxThroughput}`
```

Comment:
Since you're changing this, how about simply "Value must be between...", which seems much more natural.